docs/advanced_concepts.md

```diff
@@ -44,7 +44,7 @@ interactive table.
 ```
 
 ```Python
-dataframe = numpy.random.randn(10, 20)
+dataframe = np.random.randn(10, 20)
```

Comment:
Hah, I didn't notice before that variable name doesn't express its content well.